cmd/esc/cli/env_open.go

```diff
@@ -134,22 +145,95 @@ func renderValue(
 
 }
 
-func getEnvironmentVariables(env *esc.Environment) []string {
-	vars, ok := env.Properties["environmentVariables"].Value.(map[string]esc.Value)
-	if !ok {
-		return nil
-	}
+type prepareOptions struct {
+	quote   bool
+	pretend bool
```

Comment:
Is this the same as "dry run"?  I'd suggest renaming or adding a comment to make the intent clearer.

Reply:
Fixed